pr-template.md

```diff
@@ -46,6 +46,7 @@
 - [ ] I don’t believe this work introduces unnecessary technical debt.
 - [ ] +1 from me!
 
-#### Software Tester Selenium review:
-- [ ] I’ve run all the tests locally and they pass.
-- [ ] I agree with the test coverage.
\ No newline at end of file
+#### Software Tester Review
+- [ ] I’ve run all the selenium tests locally and they pass.
+- [ ] I agree with the test coverage.
+- [ ] I've checked for Ruby changes and understand how to deploy if present. 
```

Comment:
This is a good point but I don't think it belongs in the PR template, the software tester review can be completed by any tester not necessarily the one that will be deploying and testing the work.